package.json

```diff
@@ -29,14 +26,5 @@
             "components/supervisor/frontend",
             "charts/"
         ]
-    },
-    "resolutions": {
```

Comment:
@akosyakov I no longer see the need for these resolutions, esp. `nan`/`oniguruma` and the `react` ones.
Is it fine to remove `vscode-*` here as well? I saw them being used in `supervisor` frontend but assume they are pulled from `gitpod-protocol`, correct?

Reply:
yes, I think they should go